devenv/README.md

```diff
@@ -1,37 +1,37 @@
-This folder contains useful scripts and configuration for...
+# Set up your development environment
 
-* Configuring dev data sources in Grafana
-* Configuring dev & test scenarios dashboards.
-* Creating docker-compose file with DBs and fake data.
+This folder contains useful scripts and configuration so you can:
 
+* Configure data sources in Grafana for development.
+* Configure dashboards for development and test scenarios.
+* Create docker-compose file with databases and fake data.
 
-# Dev dashboards and data sources
+
+## Developer dashboards and data sources
 
 ```bash
 ./setup.sh
 ```
 
-After restarting grafana server there should now be a number of data sources named `gdev-<type>` provisioned as well as
-a dashboard folder named `gdev dashboards`. This folder contains dashboard & panel features tests dashboards. 
-
-#### Dev dashboards
+After restarting the Grafana server, there should be a number of data sources named `gdev-<type>` provisioned as well as
+a dashboard folder named `gdev dashboards`. This folder contains dashboard and panel features tests dashboards. 
 
-Please update these dashboards or make new ones as new panels & dashboards features are developed or new bugs are
+Please update these dashboards or make new ones as new panels and dashboards features are developed or new bugs are
 found. The dashboards are located in the `devenv/dev-dashboards` folder. 
 
-# docker-compose with databases
+## docker-compose with databases
 
```

Comment:
I am not sure if it belongs here, but maybe I would add at the beginning similar intro as you've used for macOS guide:
`Grafana uses [Docker](https://docker.com) to make the task of setting up databases a little easier. Make sure you [install Docker](https://docs.docker.com/docker-for-mac/install/) before proceeding to the next step.`